README.md

```diff
@@ -88,8 +91,8 @@ during a cache hit:
 ```mermaid
 sequenceDiagram
     actor A as Developer
-    participant B as Github
-    participant C as Github Actions
+    participant B as GitHub
+    participant C as GitHub Actions
     participant D as Turbo Cache Server
     participant E as S3 bucket
     A->>+B: Push new commit to GH.<br>Trigger PR Checks.
```

Comment:
Nice. I was also thinking about using [details/summary](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) tags here to toggle the diagram. wdyt? I it would render a shorter README, but still allow users to expand.

Reply:
I actually don't mind it at all. There is always the table of contents if you need to skip ahead.